opensrp-giz-malawi/src/test/java/org/smartregister/giz/dao/ReportDaoTest.java

```diff
@@ -0,0 +1,161 @@
+package org.smartregister.giz.dao;
+
+import net.sqlcipher.MatrixCursor;
+import net.sqlcipher.database.SQLiteDatabase;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.smartregister.giz.domain.EligibleChild;
+import org.smartregister.giz.domain.VillageDose;
+import org.smartregister.immunization.domain.Vaccine;
+import org.smartregister.immunization.repository.VaccineRepository;
+import org.smartregister.repository.EventClientRepository;
+import org.smartregister.repository.Repository;
+
+import java.text.ParseException;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.Mockito.doReturn;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReportDaoTest extends ReportDao {
```

Comment:
Update test to follow the convention `test[MethodName][ExpectedResult][Parameters] (Parameters optional if not present)`

Reply:
Done